app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -153,13 +164,15 @@ private fun RecentlyVisitedHistoryGroup(
         ) {
             RecentlyVisitedTitle(
                 text = recentVisit.title,
-                modifier = Modifier.padding(top = 7.dp, bottom = 2.dp)
+                modifier = Modifier
+                    .padding(top = 7.dp, bottom = 2.dp)
+                    .weight(1f)
             )
 
-            RecentlyVisitedCaption(recentVisit.historyMetadata.size)
+            RecentlyVisitedCaption(recentVisit.historyMetadata.size, Modifier.weight(1f))
```

Comment:
Let's add the named parameters here since we do that in most places. It makes it easier to review in GitHub since you don't have the benefits of Android Studio showing you the named parameter hints.